index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header class="header">
+            <nav>
+            <p class="time"> 9:14 <i class='fas fa-wifi'></i></p>
+            <button class="button"><a href="#">Logo</a></button> 
+            <a href="#"><i class="fa fa-bars"></i></a>
+            </nav>
+         </header>
+        </div>
+        <div class="item2">
+          <section>
+            <h1 class="mariane">I'm Mariane<br>
+              Glad to see<br>
+             you!<br>
+           </h1>
+          </section>
+            <section>
+            <p class="text">
+              I’m a software developer! I can help you<br>
+               build a product , feature or website Look <br>
+               through some of my work and<br>
+                experience! If you like what you see and <br>
+                have a project you need coded, don’t<br>
+                 hestiate to contact me.
+            </p>
+          </section>
+          <section class="cap">
+            <button>LET’S CONNECT</button>
+          </section>
+          
+<section class="img">
+  <ul>
+    <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+    <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+    <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+    <li><a href="#"><i class='fab fa-github'></i></a></li> 
+    <li><a href="#"><i class='fab fa-angellist'></i></a></li> 
+  </ul>
```

Comment:
- [ ] Kindly align the social media icons a little to the left, making use of the correct sizes for the social media icons and the correct padding and margins so that it matches the provided Figma design.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/198292310-5c6b41cb-48c5-42d3-8eb8-8dbdc6bc2832.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198292411-fb9333c8-f46b-4143-821b-cf7164a07e45.png)